app/templates/components/affiliate-unit.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="affiliate-unit__header">Suggested By Fans</div>
```

Comment:
Please add this to i18n.

Reply:
I was rather thinking this text might become part of our slots configuration in the JSON. But in the meantime I can add it to i18n, yes.